### PR TITLE
Android 13 Push Notification permission handling

### LINKED ISSUE #278: Android 13 Push Notification permission handling
Starting from Android 13, Push Notifications are disabled by default, if the app was installed on this Android version first time.

We need to implement askign for permission (similiar as on iOS).

https://developer.android.com/develop/ui/views/notifications/notification-permission

### PR DESCRIPTION
**Pull request by akrol95**
Closes #278

### CODE REVIEW

File: BMM.UI.Android/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
 	</application>
 	<uses-feature android:name="android.hardware.output" android:required="false" />
 	<permission android:name="org.brunstad.bmm.permission.C2D_MESSAGE" android:protectionLevel="signature" />
+	<uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```
Comment (kkuepper) [inline BMM.UI.Android/Properties/AndroidManifest.xml:34]: does this mean that with the next update ~40% of our users will see a message, asking to give the notification permission?
(That's not a bad thing. Just making sure I understand)

**Review by kkuepper — CHANGES_REQUESTED** [review]

File: BMM.UI.Android/Properties/AndroidManifest.xml (hunk 1 above)
Comment (akrol95) [reply BMM.UI.Android/Properties/AndroidManifest.xml:34]: No, it's only for new installations on Android 13 or never OS.
If you open the app on older OS, notification permission is always granted automatically.
If you use the app on Android 12 and then you update the OS to Android 13 you won't see the message too.

File: BMM.UI.Android/Application/Implementations/App/SystemSettingsService.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        }
+        
+        public async Task GoToSettings()
+        {
+            await _mvxMainThreadAsyncDispatcher.ExecuteOnMainThreadAsync(() =>
+            {
+                var context = _mvxAndroidCurrentTopActivity.Activity;
+
+                var settingsIntent = new Intent();
+                settingsIntent.SetAction(Android.Provider.Settings.ActionApplicationDetailsSettings);
```
[the remark this answers is not in the document]
Comment (akrol95) [reply BMM.UI.Android/Application/Implementations/App/SystemSettingsService.cs:29]: No

File: BMM.Core/Translation/en/main.json [hunk 3]
```diff
@@ -314,7 +314,8 @@
     "ThemeHeader": "Dark mode",
     "ThemeText": "Choose between a light and a dark theme",
     "SiriShortcutsHeader": "Siri Shortcuts",
-    "SiriShortcutsText": "Add Siri Shortcuts for BMM to your device"
+    "SiriShortcutsText": "Add Siri Shortcuts for BMM to your device",
+    "PushNotificationsGoToSettings": "Unfortunately we cannot enable Push Notifications as they are disabled within your device settings. Please allow Push Notifications within your device settings to enable this feature."
```
[the remark this answers is not in the document]
Comment (akrol95) [reply BMM.Core/Translation/en/main.json]: Done

**Review by kkuepper — APPROVED** [review]

**Pull request merged**